packages/editor/src/components/properties/BoxColliderNodeEditor.tsx

```diff
@@ -33,7 +33,7 @@ export const BoxColliderNodeEditor: EditorComponentType = (props) => {
   return (
     <NodeEditor {...props} description={t('editor:properties.boxCollider.description')}>
       <InputGroup name="Trigger" label={t('editor:properties.boxCollider.lbl-isTrigger')}>
-        <BooleanInput value={isTrigger} onChange={updateProperty(null!, 'isTrigger')} />
+        <BooleanInput value={isTrigger} onChange={updateProperty(ColliderComponent, 'isTrigger')} />
```

Comment:
for now, we don't want to put isTrigger onto the collider component, it will exist as metadata on the physics body already. updateBoxCollider already deals with this.